projects/angular/src/utils/i18n/common-strings.default.ts

```diff
@@ -56,6 +56,8 @@ export const commonStringsDefault: ClrCommonStrings = {
   datagridFilterDialogAriaLabel: 'Filter dialog',
   columnSeparatorAriaLabel: 'Column resize handle',
   columnSeparatorDescription: 'Use left or right key to resize the column',
+  numericFilterLabelFrom: 'From',
+  numericFilterLabelTo: 'To',
```

Comment:
```suggestion
  fromLabel: 'From',
  toLabel: 'To',
```

I would name these consistently with `minPlaceholder` and `maxPlaceholder`. (I don't think these need to restricted to the numeric filter.)